### PR TITLE
add altTextprop to avatar

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/Avatar/index.tsx [hunk 1]
```diff
@@ -48,8 +49,8 @@ export const Avatar: React.FC<AvatarProps> = ({
         statusNode={statusNode}
         statusPosition={statusPosition}
       >
-        {src ? (
-          <AvatarComponents.Image src={src} />
+        {src && alt ? (
```
Comment (monikaracka) [inline packages/design-system/src/components/Avatar/index.tsx:52]: I'm not sure about this solution. If I add `src`, I want to see an image. Maybe we should set the default `alt` eg. "avatar" if `alt` is undefined?

Comment (kielbasa-elp) [reply packages/design-system/src/components/Avatar/index.tsx:52]: In my opinion "alt" should be optional

File: packages/design-system/src/components/Avatar/index.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ export const Avatar: React.FC<AvatarProps> = ({
   caption,
   size = "md",
   src,
+  alt,
```
Comment (monikaracka) [inline packages/design-system/src/components/Avatar/index.tsx:12]:
```suggestion
  alt = "avatar",
```

File: packages/design-system/src/components/Avatar/index.tsx [hunk 3]
```diff
@@ -49,7 +50,7 @@ export const Avatar: React.FC<AvatarProps> = ({
         statusPosition={statusPosition}
       >
         {src ? (
-          <AvatarComponents.Image src={src} />
+          <AvatarComponents.Image src={src} alt={alt || "avatar"} />
```
Comment (monikaracka) [inline packages/design-system/src/components/Avatar/index.tsx:53]:
```suggestion
          <AvatarComponents.Image src={src} alt={alt} />
```

**Review by monikaracka — APPROVED** [review]

**Pull request merged**